src/components/Login/Login.jsx

```diff
@@ -1,20 +1,117 @@
-import React from 'react';
+import React, { useState, useEffect, useRef } from 'react';
 import Joi from 'joi';
-import Form from '../common/Form';
 import { connect } from 'react-redux';
 import { Link, withRouter } from 'react-router-dom';
+import Form from '../common/Form/Form';
 import { loginUser } from '../../actions/authActions';
 import { clearErrors } from '../../actions/errorsActions';
 
+/* export const Login = (props) => {
```

Comment:
Hey Miguel, did you mean to have this commented out and leave line 101 in?

Reply:
Hey @nchen0 those are components I am having trouble refactoring so I just left them commented for right now. Honestly I thought those were in a different branch, but I plan on getting that done next thing so that's why there's that whole section of commented new code on both the Login and the Form components.